src/AltinnCore/Runtime/Controllers/AuthenticationController.cs

```diff
@@ -44,9 +47,13 @@ public async Task<IActionResult> KeepAlive()
 
                 string token = await result.Content.ReadAsStringAsync();
 
-                if (string.IsNullOrWhiteSpace(token))
+                _logger.LogInformation($"token from platform{token}");
```

Comment:
Do not log the token or session cookie. 

7.1.1 Verify that the application does not log credentials or payment details. Session tokens should only be stored in logs in an irreversible, hashed form. (C9, C10)

Reply:
That was logged only for testing, removed it now